README.md

```diff
@@ -1,10 +1,15 @@
 # toil-cluster-infra
 Infrastructure for a Toil cluster (http://toil.ucsc-cgl.org) in AWS.
 
-## Workflow
-The workflow to provision AWS resources is done using pull requests.
-Requests using PRs provide history, gating, reviewing and an approval
-process.
+## Provisioning
```

Comment:
this doesn't seem to document how someone would provisioning a new toil cluser with a PR?

Reply:
I am leaving that for later when I have a chance to create the nested template.